test/WebSites/RazorWebSite/Controllers/EmbeddedViewsController.cs

```diff
@@ -7,6 +7,8 @@ namespace RazorWebSite.Controllers
 {
     public class EmbeddedViewsController : Controller
     {
+        public IActionResult Index() => View();
```

Comment:
This should have been there since beginning as we are trying to generate links and in case of Dispatching it fails as this action descriptor wouldn't be available.